src/open_dread_rando/custom_door_types.py

```diff
@@ -23,7 +23,7 @@
 
 @functools.cache
 def _template_read_shield(file: str):
-    with Path(__file__).parent.joinpath("templates", f"{file}.json").open() as f:
+     with templates_path().joinpath(f"{file}.json").open() as f:
```

Comment:
Looks like 5 spaces? Ruff is not complaining about mixing indentation?

Reply:
huh whoops